instance/models/server.py

```diff
@@ -449,3 +457,17 @@ def terminate(self):
                 self._status_to_unknown()
         else:
             self._status_to_terminated()
+
+    def _shutdown(self, os_server, poll_interval=10, max_wait=600):
```

Comment:
Do you think it's better to have the `max_wait` as system-wide variable to control the grace period from the settings instead of here?

Reply:
That wouldn't hurt, yep!